frontend/src/metabase/lib/urls.js

```diff
@@ -34,6 +26,16 @@ export function question(cardId, hash = "", query = "") {
     : `/question${query}${hash}`;
 }
 
+export const extractQueryParams = (query: Object): Array => {
+  return Object.entries(query).flatMap(flattenParam);
```

Comment:
I believe `flatMap` isn't implemented in a number of browsers. Something like this should work isntead

```
  return [].concat(...Object.entries(query).map(flattenParam));
```